### PR TITLE
adding navbar

### PR DESCRIPTION
**Pull request by atulbansal4444**

### CODE REVIEW

File: src/components/Header/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Container, Row, Col, Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import NavbarTabs from './NavbarTabs.jsx';
+
+const Navbar = () => {
+    return (
+        <Container fluid style={{fontSize: 18, padding: 5, backgroundColor: 'grey', top: 5}}>
```
Comment (RajatKaushal3956) [inline src/components/Header/NavBar.jsx:8]: There is no need to use of container as we are going to hide the links

**Review by RajatKaushal3956 — CHANGES_REQUESTED** [review]
Please do the suggested changes

File: src/components/Header/NavBar.jsx (hunk 1 above)
Comment (atulbansal4444) [reply src/components/Header/NavBar.jsx]: removed

**Comment by RajatKaushal3956** [comment]
Closing this PR don't need now

**Pull request closed without merging**